src/main/java/myPackage/Save.java

```diff
@@ -0,0 +1,85 @@
+package myPackage;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Scanner;
+
+
+public class Save {
+
+    public static void checkFile() throws IOException {
+        try {
+            File f = new File("data/duke.txt");
+            if (!f.createNewFile()) {
+                System.out.println("file created");
+                ;
+            } else {
+                System.out.println("save file found");
+            }
+        } catch (IOException e) {
+            System.out.println("file not found");
+        }
+    }
+
+    public static void load(String fileLocation) throws FileNotFoundException {
+        File f = new File(fileLocation);
+        Scanner s = new Scanner(f);
+        while (s.hasNext()) {
+            String str = s.nextLine();
+            if (str.length() > 0) {
+                String[] stringArray = str.split("\\|");
+                switch (stringArray[0]) {
+                    case ("T") : {
+                        TaskList.list.add(new ToDos(stringArray[2]));
+                        break;
+                    }
+                    case ("D") : {
+                        TaskList.list.add(new Deadlines(stringArray[2], stringArray[3]));
+                        break;
+                    }
+                    case ("E") : {
+                        TaskList.list.add(new Events(stringArray[2], stringArray[3]));
+                        break;
+                    }
+                    default: {
+                        System.out.println("wait your input is wrong");
+                    }
+                }
+            }
+
+        }
+    }
+
+    public static void save() {
+        try {
+            Path filePath = Paths.get("data/duke.txt");
+            String s = "";
+            for (int i = 0; i < TaskList.list.size(); i++) {
+                s += TaskList.list.get(i).getTaskType() + "|" + TaskList.list.get(i).getStatusIcon() + "|"
```

Comment:
This seems like a rather complicated statement you could consider abstracting this into another method